x-pack/plugins/canvas/canvas_plugin_src/functions/common/font.js

```diff
@@ -5,55 +5,74 @@
  */
 
 import inlineStyle from 'inline-style';
+import { i18n } from '@kbn/i18n';
 import { openSans } from '../../../common/lib/fonts';
 
 export const font = () => ({
   name: 'font',
   aliases: [],
   type: 'style',
-  help: 'Create a font style',
+  help: i18n.translate('xpack.canvas.functions.fontHelpText', {
```

Comment:
add .font. namespace for all ids in this file. the same for other files. Also use .args. namespace